chaos_genius/views/kpi_view.py

```diff
@@ -61,9 +67,7 @@ def kpi():
         data = request.get_json()
         data["dimensions"] = [] if data["dimensions"] is None else data["dimensions"]
 
-        data_source = DataSource.get_by_id(data["data_source"]).as_dict
-        data_con = get_sqla_db_conn(data_source_info=data_source)
```

Comment:
Why was this deleted?

Reply:
`data_con` wasn't used, so I guess this is ok. But the line that was added instead of this has trailing spaces.